vpr/src/draw/draw.cpp

```diff
@@ -34,6 +34,13 @@
 #include "globals.h"
 #include "draw_color.h"
 #include "draw.h"
+#include "draw_basic.h"
```

Comment:
Would be good to add to the top-level comment at the top of this file: say what kinds of functions should go in this file (or say put new functions in helper files if at all possible).